Flazzy/Tags/DefineBitsJPEG3.cs

```diff
@@ -29,6 +29,11 @@ public DefineBitsJPEG3(HeaderRecord header, FlashReader input)
                 int partialLength = (2 + 4 + alphaDataOffset);
                 AlphaData = input.ReadBytes(Header.Length - partialLength);
             }
+            else if (Format == ImageFormat.PNG)
+            {
+                const int EMPTY_DATA_COMPRESSED_LENGTH = 8;
+                AlphaData = input.ReadBytes(EMPTY_DATA_COMPRESSED_LENGTH);
```

Comment:
This length matches the [`partialLength` calculation](https://github.com/ArachisH/Flazzy/blob/c67ca5884faa736a42e413154e35f32dfc7799c9/Flazzy/Tags/DefineBitsJPEG3.cs#L29). Should we just count on that?